### PR TITLE
SWC-6334

### PR DESCRIPTION
**Pull request by jay-hodgson**
Likely lesson learned from portals, use bitmap image instead of svg

### CODE REVIEW

**Comment by jay-hodgson** [comment]
I'm having fun using Inkscape these days!

File: src/main/webapp/Portal.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
     />
     <meta
       property="og:image"
```
Comment (tschaffter) [inline src/main/webapp/Portal.html:28]:
```suggestion
      name="image"
      property="og:image"
```

**Review by tschaffter — APPROVED** [review]
Only one comment, otherwise LGTM.

File: src/main/webapp/Portal.html (hunk 1 above)
Comment (jay-hodgson) [reply src/main/webapp/Portal.html:28]: You're too fast at reviewing :)
Change made.

**Review by tschaffter — APPROVED** [review]

**Pull request merged**